src/components/utilities.js

```diff
@@ -28,12 +28,34 @@ export const Utilities = {
   },
 
   async getSupportedVersions() {
-    try {
-      const resp = await fetch(process.env.SUPPORTED_VERSIONS_JSON);
-      const data = await resp.json();
-      return { ...data.supportedVersions };
-    } catch (error) {
-      throw error;
-    }
+    const internalAddonVersions = fetch(process.env.SUPPORTED_VERSIONS_JSON)
+        .then(response => response.json())
+        .then(response => response.supportedVersions);
+
+    const externalAddonVersions = fetch("https://kurl-sh.s3.amazonaws.com/external/addon-registry.json")
+        .then(response => response.json());
+
+    return this.mergeAddonVersions(...await Promise.all([internalAddonVersions, externalAddonVersions]));
+  },
+
+  mergeAddonVersions(internalAddonVersions, externalAddonVersions) {
+    const addons = {};
+    Object.keys(externalAddonVersions).forEach(externalAddonName => {
+      const fileName = externalAddonName.slice(0, externalAddonName.length - 7);
+      const [name, version] = fileName.split("-");
```

Comment:
minio has version names like "2022-08-02T23-59-16Z"

Reply:
rather than encode the addon name and version in the file why not give this json more structure?

https://github.com/replicatedhq/kURL/blob/648ea625b51507f53c255f0acff669e11e128cde/.github/actions/import-external-addons/index.js#L30

Reply:
If we want to go that route, we will have to kinda go back to square one with most of these changes.

Reply:
what else would need to be changed other than the file i linked?

Reply:
The logic for the kurl backend api that was merged and obviously this pr